source/includes/common-installation.rst

```diff
@@ -211,7 +211,36 @@ MinIO **does not** support arbitrary migration of a drive with existing MinIO da
 
 .. end-local-jbod-single-node-desc
 
+.. start-storage-requirements-desc
+
+The following requirements summarize the :ref:`minio-hardware-checklist-storage` section of MinIO's hardware recommendations:
+
+Use Local Storage
+   Direct-Attached Storage (DAS) has significant performance and consistency advantages over networked storage (NAS, SAN, NFS).
+   MinIO strongly recommends flash storage (NVMe, SSD) for primary or "hot" data.
+
+Use XFS-Formatting for Drives
+   Deployments using non-XFS filesystems (EXT4, BTRFS, ZFS) tend to have lower performance and may also exhibit unexpected or undesired behavior at scale or under load.
+
+Use Consistent Type of Drive
+   Each :term:`pool` must use the same type (NVMe, SSD)  of drive with identical capacity (e.g. ``N`` TB) . 
```

Comment:
```suggestion
   Each :term:`pool` must use the same type (NVMe, SSD)  of drive with identical capacity (for example, all ``N`` TB) . 
```

I'd also replace N with a number, not sure what is a current typical size for our audience.

Reply:
Let me know how this reads.